src/app/containers/Video/fixtureData.jsx

```diff
@@ -0,0 +1,187 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import VideoContainer from '.';
+import { RequestContextProvider } from '../../contexts/RequestContext';
+
+const videoBlock = [
+  {
+    model: {
+      blocks: [
+        {
+          model: {
+            copyrightHolder: '@twitterhandle',
+            height: 549,
+            locator: 'e49c/test/a0b28560-50a9-11e9-8100-2defb9ff0749.jpg',
+            originCode: 'cpsdevpb',
+            width: 976,
+          },
+          type: 'rawImage',
+        },
+        {
+          model: {
+            blocks: [
+              {
```

Comment:
To simplify this fixture data file, you can use block helper methods such as `singleTextBlock` and pass in the text value to it. e.g.
`singleTextBlock('The residents of Ligsalz8 believe they have found the answer to the expensive cost of renting in Munich')`;